src/reactjs_jsx_ppx_v4.ml

```diff
@@ -0,0 +1,1455 @@
+("crossOrigin", {jsString = None; typeString = mkPropConst (Lident "string")});
+("cols", {jsString = None; typeString = mkPropConst (Lident "int")});
+("colSpan", {jsString = None; typeString = mkPropConst (Lident "int")});
+("content", {jsString = None; typeString = mkPropConst (Lident "string")});
+("controls", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("coords", {jsString = None; typeString = mkPropConst (Lident "string")});
+("data", {jsString = None; typeString = mkPropConst (Lident "string")});
+("dateTime", {jsString = None; typeString = mkPropConst (Lident "string")});
+("default", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("defer", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("disabled", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("download", {jsString = None; typeString = mkPropConst (Lident "string")});
+("encType", {jsString = None; typeString = mkPropConst (Lident "string")});
+("form", {jsString = None; typeString = mkPropConst (Lident "string")});
+("formAction", {jsString = None; typeString = mkPropConst (Lident "string")});
+("formTarget", {jsString = None; typeString = mkPropConst (Lident "string")});
+("formMethod", {jsString = None; typeString = mkPropConst (Lident "string")});
+("headers", {jsString = None; typeString = mkPropConst (Lident "string")});
+("height", {jsString = None; typeString = mkPropConst (Lident "string")});
+("high", {jsString = None; typeString = mkPropConst (Lident "int")});
+("href", {jsString = None; typeString = mkPropConst (Lident "string")});
+("hrefLang", {jsString = None; typeString = mkPropConst (Lident "string")});
+("htmlFor", {jsString = None; typeString = mkPropConst (Lident "string")});
+("httpEquiv", {jsString = None; typeString = mkPropConst (Lident "string")});
+("icon", {jsString = None; typeString = mkPropConst (Lident "string")});
+("inputMode", {jsString = None; typeString = mkPropConst (Lident "string")});
+("integrity", {jsString = None; typeString = mkPropConst (Lident "string")});
+("keyType", {jsString = None; typeString = mkPropConst (Lident "string")});
+("kind", {jsString = None; typeString = mkPropConst (Lident "string")});
+("label", {jsString = None; typeString = mkPropConst (Lident "string")});
+("list", {jsString = None; typeString = mkPropConst (Lident "string")});
+("loop", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("low", {jsString = None; typeString = mkPropConst (Lident "int")});
+("manifest", {jsString = None; typeString = mkPropConst (Lident "string")});
+("max", {jsString = None; typeString = mkPropConst (Lident "string")});
+("maxLength", {jsString = None; typeString = mkPropConst (Lident "int")});
+("media", {jsString = None; typeString = mkPropConst (Lident "string")});
+("mediaGroup", {jsString = None; typeString = mkPropConst (Lident "string")});
+("method", {jsString = None; typeString = mkPropConst (Lident "string")});
```

Comment:
According to [WHATWG HTML living standard](https://html.spec.whatwg.org/#attr-fs-method), this should be an enumerated attribute.
```suggestion
("method", {jsString = None; typeString = mkPropVariant ["get"; "post"; "dialog"]});
```